src/java.desktop/share/classes/java/beans/Beans.java

```diff
@@ -64,6 +64,22 @@ public Beans() {}
      * <p>
      * Instantiate a JavaBean.
      * </p>
+     * The bean is created based on a name relative to a class-loader.
+     * This name should be a {@linkplain ClassLoader##binary-name binary name} of a class such as "a.b.C".
+     * <p>
+     * The given name can indicate either a serialized object or a class.
+     * We first try to treat the {@code beanName} as a serialized object
+     * name then as a class name.
+     * <p>
+     * When using the {@code beanName} as a serialized object name we convert the
+     * given {@code beanName} to a resource pathname and add a trailing ".ser" suffix.
+     * We then try to load a serialized object from that resource.
+     * <p>
+     * For example, given a {@code beanName} of "x.y", {@code Beans.instantiate} would first
+     * try to read a serialized object from the resource "x/y.ser" and if
+     * that failed it would try to load the class "x.y" and create an
+     * instance of that class.
```

Comment:
```suggestion
     * For example, given a {@code beanName} of {@code "x.y"}, {@code Beans.instantiate} would first
     * try to read a serialized object from the resource {@code "x/y.ser"} and if
     * that failed it would try to load the class {@code "x.y"} and create an
     * instance of that class.
```
I think the example names of the classes and files should be marked up with `{@code}`. This also applies to `".ser"` in the paragraph above.